index.html

```diff
@@ -1,19 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<title>Karma</title>
+	<link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" href="css/normalize.css">
+	<link rel="stylesheet" href="css/style.css">
+	<link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+	<!-- Header/Navbar -->
+	<header class="header">
+		<a href="#">
+			<img class="header__logo" src="img/karma-logo.svg" alt="Logo for Karma">
+		</a>
+		<nav>
+			<ul class="nav__list">
+				<li class="nav__item">
+					<a class="nav__link" href="#">Meet Karma</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__link" href="#">How it Works</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__link" href="#">Store</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__link" href="#">Blog</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__link" href="#">Help</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__link" href="#">Login</a>
+				</li>
+			</ul>
+		</nav>
+		<a class="nav__mobile"><img src="img/menu-hamburger.svg"></a>
+	</header>
+
+	<!-- Page content -->
+	<div class="container">
+		<main class="main">
+			<h1>Introducing Karma</h1>
+			<h2>Bring WiFi with you, everywhere you go.</h2>
+			<button class="btn">Learn More</button>
+		</main>
+
+		<section>
+			<h2 class="heading__underMainImage">Everyone needs a little Karma.</h2>
+			<div class="service__section">
+				<section class="service">
+					<img class="service__thumbnail" src="img/icon-devices.svg" alt="Image of multiple devices">
+					<p class="service__summary">Internet for all devices</p>
+				</section>
+				<section class="service">
+					<img class="service__thumbnail" src="img/icon-coffee.svg" alt="Image of a coffee cup">
+					<p class="service__summary">Boost your productivity</p>
+				</section>
+				<section class="service">
+					<img class="service__thumbnail" src="img/icon-refill.svg" alt="Image of a device charging">
+					<p class="service__summary">Pay as You Go</p>
+				</section>
+			</div>
+		</section>
 
+		<section class="advert__section">
```

Comment:
This section overflows a little bit on the X-Axis! See https://github.com/CodeYourFuture/HTML-CSS-Coursework-Week2/pull/293/files#r752585609